docs/readthedocs/source/doc/LLM/Quickstart/fastchat_quickstart.md

```diff
@@ -0,0 +1,90 @@
+# Serving using IPEX-LLM and FastChat
+
+FastChat is an open platform for training, serving, and evaluating large language model based chatbots. You can find the detailed information at their [homepage](https://github.com/lm-sys/FastChat).
+
+IPEX-LLM can be easily integrated into FastChat so that user can use `IPEX-LLM` as a serving backend in the deployment.
+
+## Quick Start
+
+This quickstart guide walks you through installing and running `FastChat` with `ipex-llm`.
+
+## 1. Install IPEX-LLM with FastChat
+
+To run on CPU, you can install ipex-llm as follows:
+
+```bash
+pip install --pre --upgrade ipex-llm[serving, all]
+```
+
+To add GPU support for FastChat, you may install **`ipex-llm`** as follows:
+
+```bash
+pip install --pre --upgrade ipex-llm[xpu,serving] --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
```

Comment:
when installing ipex-llm serving it will install Fastchat? Should we also specify the fastchat version (e.g. >x.x ) as user may prefer their own version of FastChat.

Reply:
When installing ipex-llm[serving], fastchat will be installed as the latest version.